CHANGELOG.md

```diff
@@ -1,3 +1,92 @@
+# 94.1.0
+
+This minor release intoduces new APIs to FlexibleHeader and Ripple, and updates the MDCSnackbar appearance to the latest style. The release also includes bug fixes for multiple components.
+
+## New Features
+
+### FlexibleHeader
+
+FlexibleHeader can now inform the receiver that its view's tracking scroll view has changed through the flexibleHeaderView:didChangeTrackingScrollViewAnimated: method. Example:
+
+**Swift**
+
+```swift
+class MyAnimationDelegate: NSObject, MDCFlexibleHeaderViewAnimationDelegate {
+	func flexibleHeaderView(_ flexibleHeaderView: MDCFlexibleHeaderView,
+                          didChangeTrackingScrollViewAnimated animated: Bool) {
+   if animated {
+      // success
```

Comment:
I don't believe the `animated` flag is the success flag. Am I misreading this API?

Reply:
Done